x-pack/plugins/security_solution/public/common/components/exceptions/types.ts

```diff
@@ -49,3 +50,46 @@ export interface ExceptionsPagination {
   totalItemCount: number;
   pageSizeOptions: number[];
 }
+
+export interface FormattedBuilderEntryBase {
```

Comment:
These new types are to deal with the empty states of the entries in the builder, since the exception item entries can be in what the `ExceptionListItemSchema` would pick up as invalid. But when bubbling up the exceptions to the parent, we do check to strip out any empties.